src/modules/calculateRandomNumber.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  /**
+   * Calculate a random 4-digit number that does not start with 0
+   * and does not contain any duplicate digits.
+   *
+   * @return {number} A random 4-digit number
+   */
+  calculateRandomNumber: () => { /* Write your code here */ },
+};
```

Comment:
Usualy we create a function and then import it

```suggestion
/**
 * Calculate a random 4-digit number that does not start with 0
 * and does not contain any duplicate digits.
 *
 * @return {number}
 */
function calculateRandomNumber() => {
  /* Write your code here */
}

module.exports = { calculateRandomNumber };
```